tests/test_client.py

```diff
@@ -32,7 +32,7 @@ def json():
 
 class TestAppboyClient(unittest.TestCase):
     def setUp(self):
-        self.client = AppboyClient(api_key='API_KEY')
+        self.client = BrazeClient(app_group_id='APP_GROUP_ID')
```

Comment:
Again we're using `api_key` now.

Reply:
oh, did the rename before rebasing over the latest master and then rebased. Will update.